test_helpers/index.ts

```diff
@@ -0,0 +1 @@
+export * from './table_matcher';
```

Comment:
When Github displays a nice big red circle like this, it's one of the key indicators that you did not do a review of your own before submitting a PR. Do what you can to catch obvious issues before submitting for review.